helpers/CMakeLists.txt

```diff
@@ -6,3 +6,7 @@ include_directories(..)
 file(GLOB SOURCES "*.c")
 
 add_library(helpers STATIC ${SOURCES})
+
+if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+    include_directories(helpers/macos)
```

Comment:
could you rename `helpers/macOS/endian.h` -> `helpers/macos/endian.h`?

Reply:
yep just renamed

Reply:
the cmake build throws up the same error as mentioned in issue #60, so the cmake build isn't working

Reply:
okay solved that issue, just have to change the cmake to `include_directories(macos) `